sandbox/src/mocks/convertAdviceAndGuidanceToReferral/responses/WithAttachments.json

```diff
@@ -46,7 +46,20 @@
         "system": "https://fhir.nhs.uk/Id/ods-organization-code",
         "value": "R69"
       }
-    }
+    },
+    {
+	"url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ReferralState-1",
+	"extension": [
+           {
+		"url": "state",
+		"valueCoding": {
+			"system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralState-1",
+			"code": "NOT_BOOKED",
+			"display": "Not Booked"
+	        }
```

Comment:
The formatting looks to be a bit strange here also, it looks like there's a few other files beneath this one which also have some odd formatting.